src/blockchain/log-processors/designated-report-submitted.ts

```diff
@@ -3,7 +3,35 @@ import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../types";
 
 export function processDesignatedReportSubmittedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
-    console.log("TODO: DesignatedReportSubmitted");
-    console.log(log);
-    callback(null);
+    // @achapman: "The designated reporter purchases Stake Tokens in order to do the Designated Report"
+    // Should designatedReportSubmitted include StakedToken address?
+
+    // for phase, sqlite lacks an ENUM, but the idea here is that:
+    //  phase:null = no report submitted
+    //  phase:0 = designated report submitted
+    //  phase:1 = round1
+    //  phase:2 = round2
+    // if that seems reasonable, i'll update PR to import the enum and use name below
```

Comment:
Hm, do you mean, import the enum in TS and convert (from phase number to phase name) in the application?  IMO null, 0, 1, and 2 are reasonably self-explanatory in this context so it might not be necessary to do anything fancy!

Reply:
👍 i'm all for dumber code 😎

Reply:
I love me some dumb code! :grinning: